Attendance_app.py

```diff
@@ -7,6 +7,8 @@
 from tkinter import font as tkfont
 from calendar import monthrange
 
+from tkinter import *

```

Comment:
F403 '`from tkinter import *`' used; unable to detect undefined name
Read: https://www.flake8rules.com/rules/F403.html